mkosi

```diff
@@ -2307,7 +2360,11 @@ default_image="/boot/initramfs-{kernel}.img"
 
     # Arch still uses pam_securetty which prevents root login into
     # systemd-nspawn containers. See https://bugs.archlinux.org/task/45903.
-    disable_pam_securetty(root)
```

Comment:
Seems like a merge went wrong here. `disable_pam_securetty` should be kept instead.

Reply:
Fixed, thanks for catching this.